xmtp/README.md

```diff
@@ -13,12 +13,24 @@
 - Repeated sends of the same payload should be idempotent. When receiving a message or invite, the receiving side will store the hash of the encrypted payload alongside the decrypted result. If a message is received with an id that already exists in the DB, it is ignored.
 - We have ignored race conditions for now (as network requests may take different amounts of time). The receiver side should be tolerant of out-of-order payloads. If ordering is a must, it is possible to use multi-producer, single-consumer queues, or singleton threads for processMessages() and processPayloads() that run on an interval.
 
+### Helper database models
+
+`refresh_jobs`:
+
+```sql
+CREATE TABLE IF NOT EXISTS refresh_jobs (
```

Comment:
I'll remove this from the README once the table actually exists. Just figured it was helpful for clarity now.